dataprofiler/profilers/profiler_options.py

```diff
@@ -264,6 +268,8 @@ def __init__(self):
         self.max = BooleanOption(is_enabled=True)
         self.sum = BooleanOption(is_enabled=True)
         self.variance = BooleanOption(is_enabled=True)
+        self.num_zeros = BooleanOption(is_enabled=False)
```

Comment:
For TextOptions, these don't necessarily make sense, or at least negative doesn't make sense. Let's add to validate a check to ensure that these Options aren't changed from False.

Reply:
validate will be in a separate PR